pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "interlocking"
+version = "0.1.1"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+#yaramo = { git = "https://github.com/simulate-digital-rail/yaramo" }
+#planpro-exporter = { git = "https://github.com/simulate-digital-rail/planpro-exporter" }
+#planpro-importer = { git = "https://github.com/simulate-digital-rail/planpro-importer" }
+#railway-route-generator = { git = "https://github.com/simulate-digital-rail/railway-route-generator" }
+yaramo = { git = "https://github.com/bp2022-ap1/yaramo", branch="refactor/replace-connected-nodes-with-edges" }
+planpro-exporter = { git = "https://github.com/bp2022-ap1/planpro-exporter", branch = "refactor/replace-connected-nodes-edges" }
+planpro-importer = { git = "https://github.com/bp2022-ap1/planpro-importer", branch = "refactor/replace-connected-nodes-edges" }
+railway-route-generator = { git = "https://github.com/bp2022-ap1/railway-route-generator", branch = "refactor/replace-connected-nodes-edges" }
```

Comment:
The last three dependencies are only test dependencies, right? And the PP exporter is not necessary at all, or did I miss something? @Lietze